code/game/objects/items.dm

```diff
@@ -528,15 +537,30 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 		if(!I.damtype == BRUTE)
 			attackforce = (attackforce / 2)//as above, burning weapons, or weapons that deal other damage type probably dont get force from physical power
 		attackforce = (attackforce * I.attack_weight)
+		if(isliving(I.loc) && HAS_TRAIT(owner, TRAIT_PARRY))
+			var/mob/living/L = I.loc
+			L.attackby(riposte, owner)
+			owner.visible_message("<span class='danger'>[owner] parries [L]'s [I] with the [src]!</span>")
+			playsound(src, 'sound/weapons/deflect.ogg', 75, 0)
+			attackforce = 0
 	else if(attack_type == UNARMED_ATTACK && isliving(hitby))
 		var/mob/living/L = hitby
 		attackforce = damage
+		if(HAS_TRAIT(owner, TRAIT_PARRY))
+			L.attackby(riposte, owner)
+			attackforce = 0
+			owner.visible_message("<span class='danger'>[owner] parries [L] with the [src]!</span>")
+			playsound(src, 'sound/weapons/deflect.ogg', 75, 0)
 		if(block_flags & BLOCKING_NASTY)
 			L.attackby(src, owner)
 			owner.visible_message("<span class='danger'>[L] injures themselves on [owner]'s [src]!</span>")
 	else if(isliving(hitby))
 		var/mob/living/L = hitby
 		attackforce = (damage * 2)//simplemobs have an advantage here because of how much these blocking mechanics put them at a disadvantage
+		if(HAS_TRAIT(owner, TRAIT_PARRY))
+			L.attackby(riposte, owner)
+			owner.visible_message("<span class='danger'>[owner] parries [L] with the [src]!</span>")
+			playsound(src, 'sound/weapons/deflect.ogg', 75, 0)
```

Comment:
`deflect.ogg` is a bad soundfile. It only fits for swords.

Reply:
got a better alternative that works for a good parry sound from a design perspective?